graphdatascience/tests/integration/test_remote_graph_ops.py

```diff
@@ -73,7 +73,7 @@ def test_remote_write_back_node_similarity(gds_with_cloud_setup: AuraGraphDataSc
         G, writeRelationshipType="SIMILAR", writeProperty="score", similarityCutoff=0
     )
 
-    assert result["relationshipsWritten"] == 4
+    assert result["relationshipsWritten"] == 2
```

Comment:
This is likely changed due to a fix for node similarity export on the GDS side: https://github.com/neo-technology/graph-analytics/pull/9636